include/mbgl/shaders/webgpu/shader_program.hpp

```diff
@@ -0,0 +1,173 @@
+#pragma once
+
+#include <mbgl/shaders/shader_program_base.hpp>
+#include <webgpu/webgpu.h>
+#include <mbgl/gfx/gfx_types.hpp>
+#include <mbgl/gfx/attribute.hpp>
+#include <mbgl/gfx/color_mode.hpp>
+#include <mbgl/gfx/depth_mode.hpp>
+#include <mbgl/gfx/stencil_mode.hpp>
+#include <string>
+#include <unordered_map>
+#include <array>
+#include <optional>
+
+namespace mbgl {
+namespace shaders {
+
+#ifndef MBGL_SHADERS_PROGRAM_COMMON_TYPES_DEFINED
```

Comment:
We're using `MLN_` as a prefix for defines.

Why is this `#define` required? Is it possible to get rid of it?

Reply:
They are gone now.